WalletWasabi.Backend/Controllers/WabiSabiController.cs

```diff
@@ -167,9 +168,9 @@ public HumanMonitorResponse GetHumanMonitor()
 	[ProducesResponseType(200)]
 	public IActionResult GetUnconfirmedCoinjoins()
 	{
-		IEnumerable<string> unconfirmedCoinJoinString = GetUnconfirmedCoinJoinCollection().Select(x => x.ToString());
+		List<string> unconfirmedCoinJoinString = GetUnconfirmedCoinJoinCollection().Select(x => x.ToString()).ToList();
```

Comment:
So this is NOT necessary, but I initially made it to specify that the ordering shouldn't change. Because this fix is not really important, I ditched the idea and simply continued to use `Enumerable` to produce less diff.